classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```

Comment:
- [ ] Kindly make the `students` attribute have a getter method. `attr_reader :students`. 👍

Reply:
👍🏽